lib/sen_tweet_web/controllers/stream_controller.ex

```diff
@@ -4,4 +4,13 @@ defmodule SenTweetWeb.StreamController do
   def index(conn, _params) do
     render(conn, "index.html", streams: Bitfeels.all_streams())
   end
+
+  def new(conn, _params) do
+    render(conn, "new.html")
+  end
+
+  def create(conn, %{"user" => user, "track" => track, "filter_level" => filter_level}) do
```

Comment:
The logged in user is the one creating the streams. Pull the user from the `conn` rather than the `params`.